rental.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  attr_accessor :date
+
+  def initialize(date, book, student)
+    @date = date
+    @book = book
+    book.visits.push(self)
+    @student = student
+    student.visits.push(self)
+  end
```

Comment:
- [ ] As specified in the project requirements, Kindly make sure a rental `belongs` to a `book` and a `person` and not a student as you just tried to do.
Hint: You can achieve this by saying 
```rb
def initialize(date, book, person)
# *The rental is adding itself to the @book achieving the 'belongs-to' association
    book.rentals << self

    @person = person
    # *The rental is adding itself to the @person the 'belongs-to' association
    person.rentals << self
end
```